AEPAnalytics/Sources/Analytics.swift

```diff
@@ -121,24 +180,21 @@ public class Analytics: NSObject, Extension {
 
     /// Build analytics context data from track event data
     /// - Parameter event: an event containing track data for processing
+    /// - Parameter data: track data for processing
```

Comment:
nit: the comments should follow the Parameters: format if there are multiple parameters